core/templates/dev/head/pages/exploration_player/ProgressNavDirective.js

```diff
@@ -23,20 +23,41 @@ oppia.directive('progressNav', [
       templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
         '/pages/exploration_player/' +
         'progress_nav_directive.html'),
+      scope: {
```

Comment:
Excellent that we're introducing a scope here.

Could you move this key above templateUrl? I think we do that elsewhere.

Reply:
fixed.